dspace-server-webapp/src/test/resources/application-test.properties

```diff
@@ -14,4 +14,6 @@
 
 ## Log4j2 configuration for test environment
 ## This file is found on classpath at src/test/resources/log4j2-test.xml
-logging.config = classpath:log4j2-test.xml
\ No newline at end of file
+logging.config = classpath:log4j2-test.xml
+
+server.servlet.context-path=/
```

Comment:
Please add a comment to describe this change & why it is necessary for our test environment.  Do our tests not work when this is `/server`?